install-dev/data/xml/hook.xml

```diff
@@ -101,6 +101,11 @@
       <title>Very top of pages</title>
       <description>Use this hook for advertisement or modals you want to load first</description>
     </hook>
+    <hook id="displayBanner">
+      <name>displayBanner</name>
```

Comment:
This hook came from blockbanner module :thinking: 
I think this file must be updated but I don't know what is this file :sweat_smile: 
https://github.com/PrestaShop/PrestaShop/blob/develop/config/xml/themes/default.xml#L118

Reply:
The hook is launched in /themes/classic/templates/_partials/header.tpl
blockbanner uses this hook I guess ?

Reply:
default.xml file seems to define modules hooking at install ?

Reply:
But, the blockbanner doesn't exists anymore, it's ps_banner now :thinking:

Reply:
The default.xml file no longer seems to be used, all modules in this file are for 1.6 version.

Reply:
It's used in `prestashop:update:configuration-file-hooks-listing` Symfony command